### ISSUE TITLE
External services information

### ISSUE THREAD

**Issue by obduk** [scoutapp/roadmap#27]
It would be useful if Scout had a separate tab listing out external services (e.g. Web requests to 3rd party APIs) in a similar fashion to the Web Endpoints and Background Jobs. This information is already logged and can be viewed in individual transaction traces, however having a summary for each external endpoint with a response time graph would be useful for debugging the performance of each one. As an example NewRelic provides a similar breakdown.

**Comment by danbt79** [comment]
Yes - this is the big thing that we need NewRelic for over Scout. If I see a 3 second trace, and the controller method in question is hiding a slow external response to a service, I need to know that!

**Comment by itsderek23** [comment]
Thanks for the feedback @danbt79.

> If I see a 3 second trace, and the controller method in question is hiding a slow external response to a service, I need to know that!

Clarifying: this information should be available in a trace. Are you missing HTTP calls in traces?

**Comment by danbt79** [comment]
I'm new to Scout, admittedly - so perhaps I haven't found the ability yet (I'm not looking for tech-support here, don't worry!) but:
I know in NewRelic, when I view a list of slow queries, I can see a breakdown by element a to how many milliseconds what part that request took - and it includes data from all parts of that request - query, controller, external service (by each step of that) etc. I see tabs in Scout for things like database, but I don't see where to get a breakdown deep into the call of where the time was being eaten up and know that it's the full picture. That's what I need - maybe you have it already, that would be great.

**Comment by danielkandel** [comment]
Very useful indeed. An aggregated view for external services is one of our top requirements.

**Comment by owenbendavies** [comment]
Thank you for the update.
